<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>869</width>
         <height>653</height>
        </rect>
       </property>
       <attribute name="label">
        <string>system.methodHelp</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QListWidget" name="systemMethodHelpWidget"/>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="systemMethodHelpButton">
          <property name="text">
           <string>Request</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>869</width>
         <height>653</height>
        </rect>
       </property>
       <attribute name="label">
        <string>samurai.EventListGet</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="11" column="6">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="6">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Offset</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Limit</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>TOS (List)</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="8">
         <widget class="QTreeWidget" name="samuraiEventListGetWidget">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="autoExpandDelay">
           <number>1</number>
          </property>
          <property name="allColumnsShowFocus">
           <bool>false</bool>
          </property>
          <property name="columnCount">
           <number>4</number>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="headerCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="headerDefaultSectionSize">
           <number>550</number>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">2</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">3</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">4</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="7" column="5">
         <widget class="QDateEdit" name="periodEndDate">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QDateEdit" name="periodStartDate">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="4">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Bis</string>
          </property>
         </widget>
        </item>
        <item row="11" column="7">
         <widget class="QPushButton" name="samuraiEventListGetButton">
          <property name="text">
           <string>Request</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Von</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QCheckBox" name="periodCheck">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Zeitraum</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="6">
         <widget class="QLineEdit" name="tosLine"/>
        </item>
        <item row="1" column="2" colspan="6">
         <widget class="QLineEdit" name="offsetLine">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="6">
         <widget class="QLineEdit" name="limitLine">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="6">
         <widget class="QLineEdit" name="labelLine"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Labels (List)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="6">
         <widget class="QLineEdit" name="eventIdLine"/>
        </item>
        <item row="5" column="2" colspan="6">
         <widget class="QLineEdit" name="baseIdLine"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>EventIDs (List)</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>IncrementBaseID</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <attribute name="label">
        <string>samurai.TosListGet</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="1">
         <widget class="QPushButton" name="samuraiTosListGetButton">
          <property name="text">
           <string>Request</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QListWidget" name="samuraiTosListGetWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>periodCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>periodStartDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>periodCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>periodEndDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
